src/cli.rs

```diff
@@ -54,6 +54,9 @@ const HELP: &[(&str, &str, &str, &[&str])] = &[
         "Skip passing --features flag to `cargo` if that feature does not exist in the package",
         &[],
     ),
+    ("", "--clean-per-run", "Remove artifacts for that package before running the command", &[
+        "If used this flag with --workspace, --each-feature, or --feature-powerset, artifacts will be removed before each run.",
```

Comment:
You might want to mention that dependencies artifacts will be preserved.

Reply:
Thanks, that certainly should be mentioned.

Reply:
Done in c93bc5d